### DISCUSSION TITLE
Widget to show upcoming posts

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#363] [Feature Requests]
This could be a very useful feature, via Joe Campbell:

Show off Upcoming Posts – WordPress plugin | WordPress.org
https://wordpress.org/plugins/soup-show-off-upcoming-posts/

**Comment by stevejburge** [comment]
> Can I show schedule post on my website?
> 
> for example, I have some posts on a schedule and I want to show that on my website with the dates when it goes live.
>

**Comment by stevejburge** [comment]
One more:

> How can I let sort my posts in the frontend by expiry date? (just now i sort it with the post date, i manipulate, butt the better way was to do it by the expiration daten.)

**Comment by rizaardiyanto1412** [comment]
One more:

> I am using your plugin on a website with an Enfold theme installed. This is what I need: https://wordpress.org/support/topic/how-can-i-let-sort-my-posts-in-the-frontend-by-expiry-date/
